recipe/meta.yaml

```diff
@@ -22,17 +23,19 @@ requirements:
     - pip
   run:
     - python
-    - typing_extensions >=3.7.4.1,<4.5.0  # [py<=37]
+    - typing_extensions >=3.7.4.1,<5.0.0  # [py<=37]
```

Comment:
This requirement is not applied anywhere, because nothing satisfies `py<=37` anymore. Either remove the selector (in case typing_extensions are still needed), or remove the whole line

Reply:
Good call. I went with dropping it.